packages/contracts-bedrock/scripts/getting-started/versions.sh

```diff
@@ -48,3 +49,4 @@ echo "foundry      0.2.0 (a5efe4f)  $ver_foundry"
 echo "make         3                $ver_make"
 echo "jq           1.6              $ver_jq"
 echo "direnv       2                $ver_direnv"
+echo "just         1.29.1           $ver_just"
```

Comment:
The latest just version is 1.34.0, probably makes sense to start from the latest version

Reply:
Can we read the `versions.json` file here to avoid duplicating the version number?

Reply:
I don't want to modify this file as i believe the docs rely on it. I don't think its maintainable long term the way it is but its greater than this PR to modify this

Reply:
Thanks, we can track in a separate issue

@sbvegan for context in the tracking issue we'll create, do the docs rely on the hardcoded versions in this file?

Reply:
Generally the whole tutorial in the docs relies on this script, and I believe the kurtosis setup maintained by EF devops team also relies on these getting started scripts now

Reply:
There are too many different ways to deploy the code which creates pain as we much maintain them all, so standardizing on a single way is really important